packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/azure/appatidp.md

```diff
@@ -0,0 +1,22 @@
+1. Create an Azure app using these [instructions](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
+
+2. Skip the **Redirect URI** section for now. We come back to it later.
```

Comment:
Just want to confirm that you could actually save the app without populating a redirect uri

Reply:
Yes, I am able to save the app and go back later.